packages/core/utils/src/event-bus/index.ts

```diff
@@ -24,6 +24,15 @@ export abstract class AbstractEventBusModuleService
   abstract emit<T>(data: EventBusTypes.EmitData<T>[]): Promise<void>
   abstract emit<T>(data: EventBusTypes.Message<T>[]): Promise<void>
 
+  abstract stageEvent<TData = unknown>(
```

Comment:
suggestion: should probably support the same API as emit no?

Reply:
I believe this should be private, no?

Reply:
yeah, lets make this private